vignettes/programming_strategy.Rmd

```diff
@@ -444,6 +446,9 @@ deprecate_stop("x.y.z", "fun_xxx()", "new_fun_xxx()")
 ### END DEPRECATION
 ```
 
+For the former case above, please pass any input arguments into a call to the new function so that it still runs and pushes users towards adopting the new function.
+Consult `R/derive_var_base.R` as an example where you can see at the bottom the functions that were deprecated as a result of this new function.
```

Comment:
Could we have an example directly here rather than pointing to another file?

Reply:
yes good call!

Reply:
@thomas-neitmann take a look at my update and let me know thoughts. i'm not sure if it should really just merge with your example above now, but i personally found it useful to see exactly where this and deprecate_warn should be called as i was a little lost without opening the example code